src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/MessageType.java

```diff
@@ -0,0 +1,25 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+import java.util.Arrays;
+
+@Getter
+@RequiredArgsConstructor
+public enum MessageType {
+    PATHOLOGY_NHSACK("NHS001", "NHS Acknowledgement MIG variant number 1"),
```

Comment:
From what I've seen so far, NHSACKs for Pathology or Screening have the same NHS001 Message type, so we could change this to be more generic:

```suggestion
    NHSACK("NHS001", "NHS Acknowledgement MIG variant number 1"),
```